tests/TestHelpers/GraphHelper.php

```diff
@@ -539,6 +539,42 @@ public static function addUsersToGroup(
 		);
 	}
 
+	/**
+	 * adds multiple users to a group with invalid host in payload
+	 *
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $adminUser
+	 * @param string $adminPassword
+	 * @param string $groupId
+	 * @param array $userIds
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function addUsersToGroupWithInvalidHost(
```

Comment:
Can't we just send the host to the already present function for sending valid requests instead of rewriting the whole function?